packages/microsoft-reactnative-sampleapps/windows/SampleLibraryCS/CustomUserControlViewManagerCS.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Controls;
+
+using Microsoft.ReactNative.Managed;
+
+namespace SampleLibraryCS
+{
+    internal class CustomUserControlViewManagerCS : AttributedViewManager<CustomUserControlCS>
```

Comment:
>CustomUserControlViewManagerCS [](start = 19, length = 30)

It seems that there is a lot of boilerplate code in this class.
Can we make it simpler, remove this class, and just add custom attributes directly on the CustomUserControlCS?

Reply:
Never mind, I was not knowing how it works.

---
In reply to: [338224336](https://github.com/microsoft/react-native-windows/pull/3490#discussion_r338224336) [](ancestors = 338224336)

Reply:
No, we need to be able to support controls from any number of potential sources that we might not have source to.